apps/server/src/modules/task-card/uc/task-card.uc.ts

```diff
@@ -141,6 +110,9 @@ export class TaskCardUc {
 	async update(userId: EntityId, id: EntityId, params: ITaskCardCRUD) {
 		const user = await this.authorizationService.getUserWithPermissions(userId);
 		const card = await this.taskCardRepo.findById(id);
+		const course = params.courseId ? await this.courseRepo.findById(params.courseId) : null;
```

Comment:
additionally to above, why is there no permission check here for the new course?